frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -543,6 +543,16 @@
      :dispatch-n [[:map.layers.logic/manual]
                   [:map/popup-closed]]}))
 
+(defn remove-layer
+  [{:keys [db]} [_ layer]]
+  (let [layers (get-in db [:map :active-layers])
+        layers (vec (remove #(= % layer) layers))
+        db     (assoc-in db [:map :active-layers] layers)
```

Comment:
I guess you could do this in a single hit; `(update-in db [..] #(vec (remove....)` but it's much of a muchness.

More to the point, did we honestly not have this before?  That seems bizarre!